build/critical.go

```diff
@@ -13,4 +14,5 @@ func Critical(v ...interface{}) {
 		panic(s)
 	}
 	os.Stderr.WriteString(s)
+	debug.PrintStack()
```

Comment:
@VoidingWarranties I am under the impression that the message is most likely to get to the user's eyes if we print it to stdout instead of stderr. If `Critical` is being called, it's due to a bug in the code and we need the highest chance possible of realizing that it has been triggered.